README.md

```diff
@@ -3,11 +3,19 @@
 ## Quick Start
 
 1. Clone this repo to your local environment.
+1. Make sure you have Ruby installed by running `ruby -v`
+1. Run `bundle install` to install required gems. If you don't have Bundler installed, you can install it with `gem install bundler`
 
-2. Run `bundle install` to install required gems. If you don't have Bundler installed, you can install it with `gem install bundler`
+## Command Line Usage
 
-3. To start your server, you can run the `rackup` command. You can also start it by running `bin/start-localhost-server.sh`, but be sure to make that file executable. `chmod -x bin/start-localhost-server.sh`
+Run `./bin.convert` passing in the roman numeral you would like to convert as an argument, like so:
 
-4. Once your server is running, you can curl localhost:8000 (or whichever port your server is using), or access in the browser. Specify the /convert route and pass in the numeral you want converted, like so: `curl localhost:8000/convert?roman=XVIII`
+    `./bin/convert <numeral>`
 
-5. The server will then return a the converted numeral as a response. In this case 18.
+## API Usage
+
+To start your server, run the `rackup` command. You can also start it by running `bin/start-localhost-server.sh`, but be sure to make that file executable. `chmod -x bin/start-localhost-server.sh`
```

Comment:
It might be good for us to note that when running the `rackup` command, the default port is 9292, but when running `start-localhost-server` we made it port 8000.

Reply:
Since we are using `rack`, maybe we just get rid of the `bin/start-localhost-server.sh` in favor of `rackup`?

Reply:
I decided to get rid of the bin script, in favor of just using `rackup`, and updated the docs to reflect the default 9292 port.

Reply:
I think that's a good call.